morphapi/api/allenmorphology.py

```diff
@@ -58,13 +63,92 @@ def __init__(self, *args, **kwargs):
 
         self.downloaded_neurons = self.get_downloaded_neurons()
 
+    def get_cells(self, require_reconstruction: bool = True) -> pd.DataFrame:
+        """
+        Download the metadata for all neurons in the Allen database and save
+        it to a cells.json file.
+        """
+        cells_path = Path(
```

Comment:
Can this use Pathlib rather than os.path?

Reply:
Definitely, done now!